### PR TITLE
LG-1533, LG-2665: tooltip delay

### PR DESCRIPTION
**Pull request by shaneeza**
## ✍️ Proposed changes

[Tooltip](https://shaneeza.github.io/leafygreen-ui/?path=/story/components-tooltip--basic)
[GuideCue](https://shaneeza.github.io/leafygreen-ui/?path=/story/components-guidecue--default)

🎟 _Jira ticket:_ [LG-1533](https://jira.mongodb.org/browse/LG-1533)

### For bug fixes, new features & breaking changes

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

### For new components

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

### CODE REVIEW

**Review by audroue — APPROVED** [review]
LGTM!

File: packages/tooltip/src/Tooltip.tsx [hunk 1]
```diff
@@ -103,6 +108,11 @@ const minSize = notchWidth + 2 * borderRadius;
 const minHeightStyle = css`
   min-height: ${minSize}px;
 `;
+
+const transitionDelay = css`
```
Comment (TheSonOfThomp) [inline packages/tooltip/src/Tooltip.tsx:112]: nit: can we move the CSS to a separate file?

File: packages/tooltip/src/Tooltip.tsx [hunk 2]
```diff
@@ -103,6 +108,11 @@ const minSize = notchWidth + 2 * borderRadius;
...
+  transition-delay: ${transitionDuration.slowest}ms;
```
Comment (TheSonOfThomp) [inline packages/tooltip/src/Tooltip.tsx:113]: Should this timing be customizable?

Comment (TheSonOfThomp) [reply packages/tooltip/src/Tooltip.tsx:113]: Or at least, should the delay be optional?

Comment (shaneeza) [reply packages/tooltip/src/Tooltip.tsx:113]: we're waiting until `infoSprinkle` comes out to make this optional

**Review by TheSonOfThomp — APPROVED** [review]
🧰 💁  LGTM

**Pull request merged**